applications/app/controllers/InteractiveController.scala

```diff
@@ -66,21 +63,28 @@ class InteractiveController(
     s"$cdnPath/service-workers/$stage/$deployPath/$file"
   }
 
-  private def lookup(path: String)(implicit request: RequestHeader): Future[Either[InteractivePage, Result]] = {
+  private def lookup(
+      path: String,
+  )(implicit request: RequestHeader): Future[Either[(InteractivePage, Blocks), Result]] = {
     val edition = Edition(request)
     log.info(s"Fetching interactive: $path for edition $edition")
     val response: Future[ItemResponse] = contentApiClient.getResponse(
       contentApiClient
         .item(path, edition)
         .showFields("all")
-        .showAtoms("all"),
+        .showAtoms("all")
+        .showBlocks("all"),
```

Comment:
I suspect you could use CAPILookup.lookup for this now as it's basically the same lookup we do for any content now.